src/Network/Send/ServerType0.pm

```diff
@@ -293,6 +293,7 @@ sub new {
 		'0A46' => ['stylist_change', 'v6' ,[qw(hair_color hair_style cloth_color head_top head_mid head_bottom)]],
 		'0A49' => ['private_airship_request', 'Z16 v' ,[qw(map_name nameID)]],
 		'0A6E' => ['rodex_send_mail', 'v Z24 Z24 V2 v v V a* a*', [qw(len receiver sender zeny1 zeny2 title_len body_len char_id title body)]],   # -1 -- RodexSendMail
+		'0A76' => ['master_login', 'V Z40 a32 v', [qw(version username password_rijndael master_version)]],
```

Comment:
username not a32?

Reply:
There is no place in source where u can find a "username" as ASCII null-padded(a*).
I followed the pattern of all the other "usernames", i even followed the "old" game_login packet pattern

Reply:
0A76 always had the length 80
using 32 as size to username you can't reach the total length

Reply:
![image](https://user-images.githubusercontent.com/10372732/69968219-b5a29f00-14f8-11ea-90a6-e2dbc233c75f.png)